test/unit/routers/alpha-router/functions/best-swap-route.test.ts

```diff
@@ -44,22 +44,22 @@ import {
 const v3Route1 = new V3Route(
   [USDC_DAI_LOW, DAI_USDT_LOW, WETH9_USDT_LOW],
   USDC,
-  WETH9[1]
+  WETH9[1]!
```

Comment:
why does this need an exclamation mark?? @justindomingue is there a better way to address this typescript sensei?